docs/Сети связи/Основы сетей сотовой связи/Поколения сетей сотовой связи.md

```diff
@@ -0,0 +1,48 @@
+# Поколения сотовой связи
+
+##### __Сотовая связь__ является основой современных коммуникаций. Технически это одна из разновидностей радиосвязи, в которой абоненты связываются друг с другом с помощью сети базовых станций, принимающих и ретранслирующих сигнал от приемопередатчиков пользователей. Для того, чтобы связь была доступна везде, в _любом месте_ и _любое время_, независимо от того, где находитесь вы и ваш собеседник, таких базовых станций должно быть очень много, чтобы покрыть максимум площади и обеспечить одновременную связь сразу множеству абонентов.
+#
+#
+#
+Именно из-за карты покрытия сети этот вид связи и назвали _«сотовой»_. Все дело в том, что зоны покрытия от каждой станции немного накладываются на соседние, чтобы обеспечить непрерывность нахождения пользователя в сети. Поэтому, когда вы смотрите на схему размещения и покрытия сверху, то круги, показывающие зону действия каждой базовой станции, пересекаясь друг с другом, образуют контур, напоминающий пчелиные __соты__.
+#
+__Сотовая связь__ стала привычным явлением, поэтому сейчас сложно представить, что относительно недавно ее не было: например, в России мобильная связь начала массово распространяться только в начале XXI века. В силу того, что в России массовая сотовая связь появилась несколько позже, чем в остальном мире, у нас быстро появились сети __2G__, а сети первого поколения разворачивались не везде и проработали недолго.
+#
+Если говорить коротко, то основным отличием сотовых сетей разных поколений является __скорость передачи данных__, становившаяся все быстрее по мере развития технологий и быстродействия оборудования. Немного остановимся на особенностях каждого из поколений, всего таковых выделяют 5 разновидностей: _1G; 2G; 3G; 4G и 5G_, где __G__ означает _“generation”(поколение)_, а число означает номер этого поколения в порядке их появления. В свою очередь, поколения подразделяются на стандарты:
```

Comment:
> _1G; 2G; 3G; 4G и 5G_

[ch-md-warning-emphasis-punctuation](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-warning-emphasis-punctuation)